src/Tools/AI Test Toolkit/src/Logs/AITTestMethodLinesCompare.Page.al

```diff
@@ -0,0 +1,189 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+page 149035 "AIT Test Method Lines Compare"
+{
+    PageType = Card;
+    ApplicationArea = All;
+    SourceTable = "AIT Test Method Line";
+    ModifyAllowed = false;
+    InsertAllowed = false;
+    DeleteAllowed = false;
+
+    layout
+    {
+        area(Content)
+        {
+            group("Version Configuration")
+            {
+                Caption = 'Version Configuration';
+
+                field(Version; this.Version)
+                {
+                    Caption = 'Version';
+                    ToolTip = 'Specifies the Base version to compare with.';
+                    ApplicationArea = All;
```

Comment:
You can remove all ApplicationArea properties on controls if it matches the pages ApplicationArea since this will be inherited. Applies to all pages.